tests/test_storageclass_deletion.py

```diff
@@ -0,0 +1,71 @@
+"""
+Automates the test OCS-297:-
+1) Create a Storage Class
+2) Create a PVC
+3) Delete the corresponding storage class
+"""
+import logging
+import pytest
+
+from ocsci.testlib import tier1, ManageTest
+from ocs import constants
+from tests import helpers
+import ocs.defaults as defaults
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Deleting a Storage class
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def teardown(self):
+    """
+    Remove the resources after execution of tests
+    """
```

Comment:
empty function?

Reply:
I am deleting all the resource that i am creating in the testcase itself. Thats why i left it empty.

Reply:
Done. deleted pvc in the tear down